include/canal_internal.hrl

```diff
@@ -13,8 +13,5 @@
 % types
 
 -type req_id()  :: shackle:request_id().
--type req() :: {{pid(), reference()}, write | read} | renew.
+-type req() :: {{pid(), reference()}, write | read}.
```

Comment:
Do you need to add `reauth` to `req()`?

Reply:
Thanks for checking! Authentication is done synchronously within the server code, unlike read and write requests, so it doesn't need to be added here.